management-service/src/main/java/pl/pwr/zpi/metadata/dto/cluster/ClusterMetadataDTO.java

```diff
@@ -1,4 +1,21 @@
 package pl.pwr.zpi.metadata.dto.cluster;
 
-public record ClusterMetadataDTO(String clusterId, boolean running) {
+import lombok.Builder;
+import lombok.Data;
+import pl.pwr.zpi.reports.enums.Accuracy;
+
+@Data
+@Builder
+public class ClusterMetadataDTO {
+    String clusterId;
+    Long updatedAtMillis;
+    Accuracy accuracy;
+    boolean running;
+
+    public static ClusterMetadataDTO of(String clusterId, boolean running) {
+        return ClusterMetadataDTO.builder()
+                .clusterId(clusterId)
+                .running(running)
+                .build();
+    }
```

Comment:
Is it possible to add notification channels as well?

Reply:
Sure thing, added :D